hl7/4.2/modules/ROOT/pages/hl7-connector-studio.adoc

```diff
@@ -0,0 +1,102 @@
+= Use Anypoint Studio to Configure HL7 EDI Connector - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Studio (Studio) editors help you design and update your Mule applications, properties, and configuration files.
+
+To add and configure a connector in Studio:
+
+. <<create-mule-project,Create a Mule project>>.
+. <<add-connector-to-project,Add the connector to your Mule project>>.
+. <<configure-source,Configure a source for the connector's flow>>.
+. <<add-connector-operation,Add a connector operation to the flow>>.
+. <<configure-global-element,Configure a global element for the connector>>.
+. <<configure-other-fields,Configure the other connector fields>>.
+
+When you run the connector, you can view the app log to check for problems in real time, as described in <<view-app-log,View the App Log>>.
+
+If you are new to configuring connectors in Studio, see xref:connectors::introduction/intro-config-use-studio.adoc[Use Studio to Configure a Connector]. If, after reading this topic, you need additional information about the connector fields, see the xref:hl7-connector-reference.adoc[HL7 EDI Reference].
+
+[[create-mule-project]]
+== Create a Mule Project
+
+In Studio, create a new Mule project in which to add and configure the connector:
+
+. In Studio, select *File > New > Mule Project*.
+. Enter a name for your Mule project and click *Finish*.
+
+[[add-connector-to-project]]
+== Add the Connector to Your Mule Project
+
+Add HL7 EDI Connector to your Mule project to automatically populate the XML code with the connector's namespace and schema location and add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In the *Add Dependencies to Project* window, type `HL7` in the search field.
+. Click HL7 EDI in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a connector to a Mule project in Studio does not make that connector available to other projects in your Studio workspace.
+
+[[configure-source]]
+== Configure a Source
+
+A source initiates a flow when a specified condition is met.
+You can configure this source to use with HL7 EDI Connector:
+
+* HTTP Listener +
```

Comment:
Since you are only listing one source, it shouldn't be a list item. You can work it into the text like:
You can configure HTTP Listener as the source for the HL7 EDI Connector to initiate a flow each time the listener receives a request on the configured host and port.

Reply:
Something like that...

Reply:
Fixed.